custom-words.txt

```diff
@@ -1329,11 +1339,12 @@ PrivateLink
 privateLinkForAzureAd
 privatelinkhub
 privatelinks
+privatelinkservice
 privatepreview
 projectable
 propogation
 Protectable
-ProviderHub
+Providerhub
```

Comment:
Why changed the case?

Reply:
It had been added with both casings. CamelCase is already treated as separate words by cSpell